lib/fuzz/fuzz.go

```diff
@@ -0,0 +1,22 @@
+package fuzz
+
+import (
+	"github.com/gravitational/teleport/lib/utils"
+	"github.com/gravitational/teleport/lib/utils/parse"
+)
+
+func Fuzzutils(data []byte) int {
```

Comment:
`go-fuzz` requires the function to be named `Fuzz` exactly, without a suffix. This means that there's only 1 fuzzer per package/directory. These individual fuzzers must live in sub-directories of `lib/fuzz`.

If you want, we can merge this in and I'll move things around afterwards as needed.

Reply:
I would like to mention that the name of each harness is flexible if you opt to fuzz continuously through oss-fuzz. As a reference, check out [Prometheus' fuzzers](https://github.com/prometheus/prometheus/blob/master/promql/fuzz.go). These run fine through oss-fuzz. 
As mentioned earlier, I have the setup ready to integrate teleport into oss-fuzz, if you would like to proceed in that direction.
I am also fine with merging now and moving things afterwards.

Reply:
Ahh, I see. Please update `README.md` to point to `oss-fuzz` instead of `go-fuzz`, because you can't actually run these fuzzers locally with `go-fuzz`.

P.S. sorry for the response delay!